bin/hbase-config.sh

```diff
@@ -94,6 +94,10 @@ do
     shift
     # shellcheck disable=SC2034
     DEBUG="true"
+  elif [ "--help" = "$1" ] || [ "-h" = "$1" ]
+  then
+    echo "Error usage! You should use it like 'hbase --help' or 'hbase -h'";
```

Comment:
Can you please provide an example of what exact command will bring the control here? 
Is it `bin/hbase-config.sh --help` ?

Reply:
bin/hbase --config ./conf/ --help version

Reply:
hbase will call hbase-config, and pass the parameters

Reply:
Got it, yeah user should not provide `--help` with `--config` or any other argument. This error handling is definitely helpful. Thanks @WenFeiYi .